src/Engine/GameActionSet.java

```diff
@@ -78,7 +78,7 @@ public String toString()
     {
       // NOTE: Ability and Unit Production actions return null for getType(), since they have no associated UnitActionFactory.
       // However, they are also never put into GameActionSets, so this is probably fine.
-      return actionOptions.get(0).getType().name();
+      return actionOptions.get(0).getType().name(actionOptions.get(0).getActor());
```

Comment:
This comment may not have aged well. It says production actions aren't put into `GameActionSets`, but this change is only needed for production actions.

Reply:
The comment is referring to building units from buildings, but that's probably fair.